crates/fluvio-sc/src/controllers/mirroring/controller.rs

```diff
@@ -195,3 +223,23 @@ impl<C: MetadataItem> RemoteMirrorController<C> {
         }
     }
 }
+
+fn option_tlspolicy(home: &Home) -> Option<TlsPolicy> {
```

Comment:
TLS should mandatory

Reply:
Not mandatory.

Reply:
This is not needed anyway since SCConfig and Remote config are two different things.